prisma/schema.prisma

```diff
@@ -38,11 +38,13 @@ model Users {
 }
 
 model Calendar {
-  id        Int     @id @default(autoincrement())
-  name      String
-  ownerId   Int
-  owner     Users?  @relation(fields: [ownerId], references: [id])
-  isDeleted Boolean @default(false)
+  id         Int     @id @default(autoincrement())
+  name       String
+  ownerId    Int
+  owner      Users?  @relation(fields: [ownerId], references: [id])
+  isPrimary  Boolean @default(false)
+  isSelected Boolean @default(false)
```

Comment:
What is meaning of this `isSelected` column?

Reply:
isSelected was added to know whether the calendar is selected or not when there are multiple calendars, but since be will be showing all external calendars are stored in a different table, this is not required will remove it.